Dockerfile

```diff
@@ -1,6 +1,20 @@
 FROM ubuntu:20.04
 
-RUN apt-get update && apt-get install -y wget gcc-arm-linux-gnueabi gcc-arm-linux-gnueabihf git build-essential bison flex bc u-boot-tools
+RUN apt-get update && \
+	apt-get install -y \
+		wget \
+		gcc-arm-linux-gnueabi \
+		gcc-arm-linux-gnueabihf \
```

Comment:
This one is not used, right? Please remove

Reply:
Removed!

Reply:
As it turns out, we do need gcc-arm-linux-gnueabi and gcc-arm-linux-gnueabihf. I'm bringing them back.

Reply:
No, we don't need `gcc-arm-linux-gnueabihf` and it should be removed (and by removing I don't mean commented out, I mean removing the line)